unc_classification_tab_data/models/mlp.py

```diff
@@ -0,0 +1,252 @@
+from typing import Tuple
+
+import numpy as np
+import torch.nn as nn
+import torch
+from torch.utils.data import Dataset, DataLoader
+from .constants import *
+
+
+class MLPModule(nn.Module):
+    """Create a new multi-layer perceptron instance with ReLU activations and no non-linearity in
+    the output layer.
+
+    Parameters
+    ----------
+    model_params : dict
```

Comment:
why not use these parameters explicitely?

Reply:
Would that be better?

Reply:
yes! much better. 

- Explicit better than implicit
- Much easier docstring and signature to inspect on the fly
- You can have checks on typing from IDEs
- etc.

Reply:
done!